dk.gqrx.gqrx.yaml

```diff
@@ -0,0 +1,311 @@
+app-id: dk.gqrx.gqrx
+runtime: org.kde.Platform
+runtime-version: '5.15-24.08'
+sdk: org.kde.Sdk
+command: gqrx
+rename-icon: gqrx
+finish-args:
+  - --device=all
+  - --share=network
```

Comment:
Required for network access to the SDR hardware and for Gqrx data streaming and control over the network.